### PR TITLE
media-video/baka-mplayer: New live ebuild

### PR DESCRIPTION
**Pull request by cstew2**
Package has been updated with some bug fixes on git
but has not had a release in a while.

Signed-off-by: Christopher Steward <cstew764@gmail.com>
Package-Manager: Portage-2.3.79, Repoman-2.3.16

### CODE REVIEW

**Comment by a17r** [comment]
- There are no open bugs for this package - is there anything important that should be backported to current stable instead?
- Who is going to maintain it, do you volunteer?

**Comment by cstew2** [comment]
- Yes, the reason I made the ebuild was because I was having trouble with seeking in the 2.0.4 version which I think is related to this issue: https://github.com/u8sand/Baka-MPlayer/issues/239
- I am willing to maintain it.

**Comment by cstew2** [comment]
I tested this patch locally with mpv 0.29.1-r1 and it fixes the issue I was having with seeking not working.

**Comment by Pesa** [comment]
No objections.

File: media-video/baka-mplayer/baka-mplayer-2.0.4.ebuild [hunk 1]
```diff
@@ -31,7 +31,7 @@ DEPEND="${RDEPEND}"
 
 S="${WORKDIR}/Baka-MPlayer-${PV}"
 
-PATCHES=( "${FILESDIR}/${P}-gcc5.patch" )
+PATCHES=( "${FILESDIR}/${P}-gcc5.patch" "${FILESDIR}/${P}-mpv23.patch" )
```
Comment (a17r) [inline media-video/baka-mplayer/baka-mplayer-2.0.4.ebuild]: PS: If this is a runtime fix, then you will have to make a revbump to this package, drop stable keywords. no changes to the existing ebuild.

Comment (cstew2) [reply media-video/baka-mplayer/baka-mplayer-2.0.4.ebuild]: The patch was just for a runtime bug so I did the revbump without stable keywords and reverted the 2.0.4 ebuild.

**Comment by a17r** [comment]
Please see the following fixes of mine:
506888987aa4c725b3fb96d12028c2ccd7351fd7
4253ec1fa82a4ab97f0458b6532327d2f52a31e9
329a8cc9c8db2f59faf27eff4a8ca0c0aa505c06
0519d13f6e6b3068335cf9c7984faae2c85bf5de

**Pull request closed without merging**